os/include/tinyara/mpu.h

```diff
@@ -49,4 +49,27 @@ enum {
 #define STACKGUARD_MPU_REGION_NUM		3	/* Region number 3 is reserved for stack protection */
 #endif
 
+/* Enum for MPU region numbers */
+
+enum MPU_REG_NUM {
+#ifndef CONFIG_APP_BINARY_SEPARATION
+	MPU_REG_NUM_UFLASH,
+	MPU_REG_NUM_URAM,
+#else
+	MPU_REG_NUM_APP_TXT,
+	MPU_REG_NUM_APP_RO,
+	MPU_REG_NUM_APP_DATA,
+#ifdef CONFIG_MPU_STACK_OVERFLOW_PROTECTION
+	MPU_REG_NUM_APP_STK,
```

Comment:
MPU_REG_NUM_APP_STK, the stack overflow is not related to APP, so we can remove APP from the naming

Reply:
done